include/aspect/stokes_matrix_free.h

```diff
@@ -543,6 +617,13 @@ namespace aspect
       Table<2, VectorizedArray<double>> active_viscosity_table;
       MGLevelObject<Table<2, VectorizedArray<GMGNumberType>>> level_viscosity_tables;
 
+      Table<2, VectorizedArray<double>> active_viscosity_derivative_wrt_pressure_table;
+
+      Table<2, SymmetricTensor<2, dim, VectorizedArray<double>>> active_strain_rate_table;
+
+      Table<2, SymmetricTensor<2, dim, VectorizedArray<double>>>
+      active_viscosity_derivative_wrt_strain_rate_table;
```

Comment:
I know a lot of the other variables here are lacking documentation as well, but we have to start somewhere: Please add documentation for your new variables. The explanation of the quantities is likely similar to what you wrote above, but what does `active_...` mean in this context?